app/views/unattended/provisioning_templates/registration/remote_execution_pull_setup.erb

```diff
@@ -0,0 +1,82 @@
+<%#
+kind: snippet
+name: remote_execution_pull_setup
+model: ProvisioningTemplate
+snippet: true
+description: |
+  Snippet for installing and setting up a client to be reachable by remote execution in pull mode
+-%>
+# check if system is registered
+subscription-manager identity > /dev/null
+if [ $? -ne 0 ]; then
+    echo "Please register with subscription-manager first, then re-run '$0'."
+    exit 1
+fi
+
+set -e
+
+<%= install_packages('foreman_ygg_worker') %>
```

Comment:
What repo do I have to enable to be able to install `foreman_ygg_worker` on CentOS 7?

Reply:
To be honest I can't really tell what is the right course of action right here. Docs should always be updated

I don't think it is a good practice to enable random public repos in katello world.

Reply:
And what exactly should go there and where specifically?

Reply:
To [Managing hosts > 3. Registering Hosts](https://docs.theforeman.org/nightly/Managing_Hosts/index-foreman-el.html#Registering_Hosts_managing-hosts)

There is description of each parameter, used templates / snippets and so on ...

Reply:
https://github.com/theforeman/foreman-documentation/pull/1316